lib/board.rb

```diff
@@ -1,89 +1,48 @@
 class Board
-  attr_reader :cells
-
-  def initialize
```

Comment:
I removed this as something that needs to be initialized with Board, and instead added it as a Ruby Constant. Here's some more info on those - https://www.rubyguides.com/2017/07/ruby-constants/
Having as few instance vars as possible has been my understanding of best practice. :) It can be referenced throughout this file as `CELLS` (as defined below), and `Board::CELLS` in other files.